index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
```

Comment:
AppVeyor is CI Service, all the log will save to it's website, and show out in web page.